catalogue/webapp/components/IIIFViewer/ViewerTopBar.tsx

```diff
@@ -338,7 +338,6 @@ const ViewerTopBar: FunctionComponent<Props> = ({ viewerRef }: Props) => {
                     ariaControlsId="itemDownloads"
                     title={work.title}
                     workId={work.id}
-                    license={licenseInfo}
```

Comment:
Let's go a bit further here, and remove the `license` parameter from the Download component.

It looks like this is the only instance of the component where we pass in license information.